src/common/core/setcore.js

```diff
@@ -69,9 +78,9 @@ define(['common/util/assert'], function (ASSERT) {
                 relid = Math.floor(Math.random() * MAX_RELID);
             } while (existingRelIds.indexOf(relid) !== -1);
             return '' + relid;
-        };
+        }
 
-        var harmonizeMemberData = function (node, setName) {
+        function harmonizeMemberData(node, setName) {
```

Comment:
Thanks, noted!

I wasn't checking the function (just removed the var), but next time I am around, I will definitely refactor it.